### ISSUE TITLE
Scenes on shared device?

### ISSUE THREAD

**Issue by K-MTG** [nfarina/homebridge#724]
Hi,

Just finished setting up Homebridge for the first time with Smartthings! It works great, however on my shared device I can't get the scenes to work. Is it possible to do that? I have a scene called "Good Night" but it only runs on the admin device and iCloud on the shared device Siri says I can't do that. Is it possible to get this to work? Thanks

**Comment by K-MTG** [comment]
Appreciate a response, spent hours trying to figure this out. (: 

Thanks

**Comment by apwelsh** [comment]
I dont know a lot on this topic, but I do know a little. Is the shared device on the WIFI network along with the homebridge server? If not, you must have an Apple TV to act as the bridge between iCloud and your network.  I have not been able to get an Apple TV 3rd generation to work, even though it is supposed to, so you may have to have to latest version of Apple TV for this.

Also, I dont export my scenes, I export virtual switches that turn on/off the scenes within SmartThings, so if you are on the same WIFI and the shared device doesnt work, this may be a feasible alternative. 
I think the best results are achieved when people use homebridge as just a wrapper as it was originally intended, rather than a smart hub.  Put absolutley everything in SmartThings and only use homebridge to siri enable smartthings.  If you have a scene controller, export it (i use virtual switches because indont have scene controllers). This also helps to ensure homekit behaves exactly like smartthings.

I have one scene that turns on all my living room lights, and another that turns them all off.  So i export the the swit mch that controls the scene and all my devices (shared or otherwise) work via homekit to control the scene, since it is the switch that changes state, and it is SmartThings rules that do all the work.

**Comment by K-MTG** [comment]
Yes, the iPad is on my primary iCloud ID and the Apple TV 3rd gen is on the same ID. The iPad works on LTE and Wi-Fi, the scenes work without an issue and the same iPhone with the same iCloud as the iPad works flawlessly. 

However, I shared homekit with another family members iPhone who is only able to access home kit on the local network and cannot control the scenes I created, Siri simply says I can't do this. I downloaded the "Devices" app on the shared iPhone and all of the devices on the iPad showed up but the scenes can not be created on the shared home kit iPhone either.

Yup, I did exactly that I created a virtual switch so when Home is on the routine home runs, etc for goodbye and goodnight. I have scenes turn on these switches. 


Here is a breakdown of the issues I have so far:
1) I cannot control scenes using the shared homekit iPhone
2) The shared iPhone only works on Wi-Fi while the primary iPad works on Wi-Fi and LTE. The Apple TV 3rd gen is on the same iCloud as the primary iPad
3) I also want to intergrate a momentary switch so I can control my garages which are on a Evolve Z-Wave relay. 

Which app are you using to control home kit devices? I am using devices. I am also curious about your setup, perhaps we can discuss this as I just began using home bridge yesterday.

**Comment by apwelsh** [comment]
I dont have any homekit devices.  I only use homebridge to control switches, and other devices via homebridge-smarthings plugin.  

It looms like we have different result though, as i cannit make homekit work theough my apple tv noatter what, but all my shared devices are able to control all my switches.  Invluding my triggering my scenes via virtual switches.  On my hub, I have a rule that says, when button pressed, activate bedtime, and another rule that says when another button is pressed arm the house, etc.  homekit only sees these buttons.

**Comment by K-MTG** [comment]
So do you say "turn on goodbye" or just tell Siri "goodbye"

Goodbye works as I set it using the devices on primarily iCloud but not on the shared
